src/app/components/home.component.ts

```diff
@@ -649,24 +649,43 @@ export class HomeComponent implements OnInit, AfterViewInit {
   }
 
   ngAfterViewInit() {
-    this.computePreviewWidth(); // so that fullView knows its size
+    this.computePreviewWidth(); // so that fullView knows its size // TODO -- check if still needed!
     // this is required, otherwise when user drops the file, it opens as plaintext
     document.ondragover = document.ondrop = (ev) => {
       ev.preventDefault();
     };
     document.body.ondrop = (ev) => {
       if (ev.dataTransfer.files.length > 0) {
-        const fullPath = ev.dataTransfer.files[0].path;
+        const fullPath: string = ev.dataTransfer.files[0].path;
         ev.preventDefault();
         if (fullPath.endsWith('.vha2')) {
           this.electronService.ipcRenderer.send(
-            'load-this-vha-file', ev.dataTransfer.files[0].path, this.saveVhaIfNeeded()
+            'load-this-vha-file', fullPath, this.saveVhaIfNeeded()
           );
         }
       }
     };
   }
 
+  /**
+   * Handle dropping something over an item in the gallery
+   * Used to handle dropping a .jpg file to replace preview!
+   * @param event         drop event - containing path to possible jpg file
+   * @param galleryItem   item in the gallery over which jpg was dropped
+   */
+  droppedSomethingOverVideo(event, galleryItem: ImageElement) {
+    const pathToNewJpg: string = event.dataTransfer.files[0].path;
+    if (pathToNewJpg.endsWith('.jpg') && galleryItem.cleanName !== '*FOLDER*') {
```

Comment:
`.JPG` `.JPEG` `.jPeG`? 😄 
Can we support other image types?

Reply:
thanks! I forgot the `toLowercase()` 😅

Reply:
fixed with #256